Stellar-contract-config-setting.x

```diff
@@ -1,6 +1,13 @@
 %#include "xdr/Stellar-types.h"
 
 namespace stellar {
+// General “smart contract execution lane” settings
+struct ConfigSettingContractExecutionLanesV0
+{
+    // maximum number of “smart” transactions per ledger
```

Comment:
nit: `Soroban` would be more appropriate (we've been trying to use term `Soroban` more in core).

Reply:
sg, fixed